src/lsm/groove.zig

```diff
@@ -550,63 +610,63 @@ pub fn GrooveType(
             if (has_id) groove.ids.reset();
 
             groove.prefetch_keys.clearRetainingCapacity();
-            groove.prefetch_objects.clearRetainingCapacity();
+            groove.objects_cache.reset();
 
             groove.* = .{
                 .objects = groove.objects,
                 .ids = groove.ids,
                 .indexes = groove.indexes,
                 .prefetch_keys = groove.prefetch_keys,
-                .prefetch_objects = groove.prefetch_objects,
                 .prefetch_snapshot = null,
+                .objects_cache = groove.objects_cache,
             };
         }
 
         pub fn get(groove: *const Groove, key: PrimaryKey) ?*const Object {
```

Comment:
When `constants.verify` can we compare this against `Tree.lookup_from_memory`?

Reply:
We could; but `lookup_from_memory` which calls `.get` on the mutable table is quite expensive, it'll have to do a full sort.

Reply:
Ah nvm then, let's leave it as-is.